src/main/java/co/github/martinhermes86/Person.java

```diff
@@ -0,0 +1,50 @@
+package co.github.martinhermes86;
+
+import java.util.Objects;
+
+public class Person {
+    private String name;
+    private int age;
+
```

Comment:
Looks nice maybe you could add an address for the person?
```suggestion
private String address;
```